docs/setup/piweb_auto_importer.md

```diff
@@ -36,7 +36,7 @@ Import plug-ins are usually installed via *.pip* files to a specific plug-in ins
 ![PiWeb Auto Importer command line example](../../assets/images/setup/piweb_auto_importer/cmd_plugin_search_path.png "PiWeb Auto Importer command line example")
 
 {: .note }
-The <span class="nowrap">`-pluginSearchPaths`</span> command line parameter is only available when *PiWeb Auto Importer* is in development mode. See above for how to active development mode.
+The <span class="nowrap">`-pluginSearchPaths`</span> command line parameter is only available when *PiWeb Auto Importer* is in development mode. See [Development mode]({% link docs/setup/piweb_auto_importer.md %}#development-mode) for how to activate development mode.
 
 ## Plug-in management
 To see whether plug-ins in custom search paths are found, you can compile one of the example plug-ins, use its build output path as custom search path and open the plugin management view.

```

Comment:
"plugin management view" -> "plug-in management view"

Reply:
Fixed.